octavia/cmd/agent.py

```diff
@@ -53,7 +54,10 @@ def load_cert_chain(self, certfile, keyfile=None, password=None, ca=None):
 
     def wrap_socket(self, sock, **kwargs):
         return super(OctaviaSSLContext, self).wrap_socket(
-            sock,
+            # Known bug in Python 2.7 - http://bugs.python.org/issue13942
+            # http://stackoverflow.com/questions/16739293/
+            # python-cannot-rebuild-ssl-socket-after-putting-in-multiprocessing-queue
+            socket.socket(_sock=sock),
```

Comment:
@liaz-stratoscale  i would recommend sending them a patch .. this could "verify" that our fix is indeed a good fix